src/DIPS.Xamarin.UI.Android/InternalButtonRenderer.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.ComponentModel;
+using Android.Content;
+using Android.Views;
+using DIPS.Xamarin.UI.Android;
+using DIPS.Xamarin.UI.Internal;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+using TextAlignment = Xamarin.Forms.TextAlignment;
+
+[assembly: ExportRenderer(typeof(InternalButton), typeof(InternalButtonRenderer))]
+
+namespace DIPS.Xamarin.UI.Android
+{
+    /// <summary>
+    /// An internal class to use when having to interact with the buttons on Android platform
+    /// </summary>
+    internal class InternalButtonRenderer : ButtonRenderer
```

Comment:
To make sure that this renderer won't get stripped away from the `Linker` you will have to add a `Initialize` method that get's run when people call `Library.Initialize`. [Please have a look at this renderer initialization.](https://github.com/DIPSAS/DIPS.Xamarin.UI/blob/9cb3a5e2e7842b424c7c79628e451fdd1756a004/src/DIPS.Xamarin.UI.Android/Library.cs#L18)